### PR TITLE
feat: add option for specifying branches and tags for head tag

### PR DESCRIPTION
**Pull request by emilsivervik**

### CODE REVIEW

File: docker-gpr/action.yml [hunk 1]
```diff
@@ -21,6 +21,24 @@ inputs:
...
+  stage-branch:
+    description: Branch name for staging environment
+    default: 'development'
+    required: false
+  stage-tag:
+    description: Image tag for staging environment
+    default: 'development'
+    required: false
+  prod-branch:
+    description: Branch name for staging environment
```
Comment (danielronnkvist) [inline docker-gpr/action.yml:34]:
```suggestion
    description: Branch name for production environment
```

File: docker-gpr/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@actions/core": "^1.2.0",
-    "@actions/exec": "^1.0.1"
+    "@actions/core": "^1.2.4",
+    "@actions/exec": "^1.0.4"
```
Comment (danielronnkvist) [inline docker-gpr/package.json:14]: How come the `node_modules` folder is added and these are still set as dependencies?

Comment (emilsivervik) [reply docker-gpr/package.json:14]: As I understood it the node_modules needs to be commited into the repo in order for the action to work, I don't quite understand the second question

Comment (danielronnkvist) [reply docker-gpr/package.json:14]: Ah, well I was just confused to having a _normal_ package.json but still having the node_modules commited to the repo. But makes sence that they are needed to run the action.

**Pull request merged**